requirements.txt

```diff
@@ -8,4 +8,4 @@ distributed>=1.24.2
 dask>=1.1.0
 psutil>=5.4.8
 click>=7.0.0
-scikit-learn>=0.20.0
+scikit-learn>=0.20.0,!=0.22
```

Comment:
is sklearn only used by the `DFSWrapper`? if so, maybe we should make it an optional dependency since that functionality is "beta" anyways

Reply:
Yeah it's only used in that wrapper